tests/test_report_reader.cpp

```diff
@@ -89,14 +89,19 @@ TEST_CASE("SomaReportReader", "[base]") {
     REQUIRE(data.data == std::vector<float>{3.2f, 4.2f, 3.3f, 4.3f, 3.4f, 4.4f, 3.5f, 4.5f});
 
     auto data_all = pop.get();
-    REQUIRE(data_all.ids == DataFrame<NodeID>::DataType{{1,  2,  3,  4,  5,  6,  7,  8,  9,  10,
-                                                         11, 12, 13, 14, 15, 16, 17, 18, 19, 20}});
+    REQUIRE(data_all.ids == DataFrame<NodeID>::DataType{{10, 11, 12, 13, 14, 15, 16, 17, 18, 19,
```

Comment:
we weren't checking the data output here? I'll add a test for that (#187), so that we can check the `data_all.data` is moving around as expected.  Once that's in, can you merge and update it as required?

Reply:
Perfect, will do right away, thank you so much.

Reply:
@mgeplf I have rebased `master` and fix the test to match the correct output. One question, do you think it is worth it to also add the same test in Python? I have tried to make them consistent in this PR, but it is your call, of course.

Reply:
> do you think it is worth it to also add the same test in Python?

Yeah, good point - I spaced on that, didn't want to block you, but wanted you to have something to test.

We don't have to do that in this PR, though.

Reply:
Added! Thanks to @olupton for the tip about using `dtype=np.float32` to compare the values correctly.

@mgeplf We might consider to create a separate PR at some point to double-check the other tests that use floating point values, as right now they do approximate comparisons with [`np.testing.assert_allclose()`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html). More importantly, we should also add data tests for the `ElementReportReader`.